### DISCUSSION TITLE
GISTIC的score文件cnv有重复区域

### DISCUSSION THREAD

**Discussion by xuexingwangzhedishitian** [ShixiangWang/self-study#114] [Q&A]
翔哥好，
我想用GISTIC的scores.gistic文件来绘制cnv的圈图，但是绘制时，我发现scores.gistic文件的cnv有重复的区域，这个导致了图给每个cnv片段上色的时候报错

第二个缺失的cnv片段和第三个扩增片段有overlap
![image](https://user-images.githubusercontent.com/58718684/215658806-8f6e311a-50e7-4c8a-a7ef-e70cbfded7a3.png)

**Comment by ShixiangWang** [comment]
有重叠区域的区间需要处理 你可以使用bedtools之类的工具

另外我之前处理gistic输入文件有写过一个预处理重叠的脚本 你可以作为参考 https://github.com/ShixiangWang/install_GISTIC/blob/master/preprocess.R

**Reply by ShixiangWang** [reply]
另外数据少的情况可以手动修改下有重叠的区域坐标值
